frontend/vite.config.js

```diff
@@ -15,6 +15,6 @@ export default defineConfig({
     },
   },
   server: {
-    port: 3000,
+    port: 5173,
```

Comment:
Take care about it

Reply:
yes, thanks next time we will check before making a PR 👍